website/blog/2023-03-30-relay-15.mdx

```diff
@@ -70,7 +70,7 @@ Flow users will now get types inferred from `graphql` literals with more Relay A
 
 ### Relay Resolver improvements
 
-A significant portion of our development effort since our last release has gone into improving [Relay Resolvers](https://relay.dev/docs/guides/relay-resolvers) (a mechanism for exposing derived data in the graph). It is worth noting that Relay Resolvers are still experimental and API changes might occur in the future.
+A significant portion of our development effort since our last release has gone into improving [Relay Resolvers](https://relay.dev/api-reference/relay-resolvers/introduction/) (a mechanism for exposing derived data in the graph). It is worth noting that Relay Resolvers are still experimental and API changes might occur in the future.
```

Comment:
Is the last sentence still applicable after we release them as stable? Maybe we can remove the "Relay Resolvers are still experimental" bit

Reply:
I think it's okay for a blog post to be a point in time artifact. That said, maybe I can add and an "update" section.